bot/code_review_bot/__init__.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         self.path = path
         self.line = line
         self.nb_lines = nb_lines
-        self.check = check
+        self.check = check or "unknown"
```

Comment:
If we go for a default value, we could put it in the function definition directly.